### PR TITLE
feat: extendable scan report format

### PR DESCRIPTION
**Pull request by spalt08**
Frontend part is broken

### CODE REVIEW

File: packages/worker/src/tasks/syncPackageIndexBatch.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import { internalApi, PackageMetadata } from '@gradejs-public/shared';
+import { systemApi, SystemAPI, PackageMetadata } from '@gradejs-public/shared';
```
Comment (ctizen) [inline packages/worker/src/tasks/syncPackageIndexBatch.ts:1]: Can we use the same identifier for types and runtime? Having two identifiers with different casing looks misleading.

File: packages/public-api/src/website/service.ts [hunk 2]
```diff
@@ -106,15 +112,12 @@ export async function syncWebPageScanResult(scanReport: SystemApi.ScanReport) {
   });
 }
 
-function mapInternalWebsiteStatus(status: internalApi.WebPageScan.Status) {
+// TODO: add protected website use-case
```
Comment (zardak) [inline packages/public-api/src/website/service.ts:115]: Do you intentionally want to merge this and fix after?

**Review by zardak — APPROVED** [review]

File: packages/public-api/src/website/service.ts (hunk 2 above)
Comment (spalt08) [reply packages/public-api/src/website/service.ts:115]: Yes

Comment (spalt08) [reply packages/public-api/src/website/service.ts:115]: Task: https://github.com/fingerprintjs/gradejs-backend/issues/195

File: packages/worker/src/tasks/syncPackageIndexBatch.ts (hunk 1 above)
Comment (spalt08) [reply packages/worker/src/tasks/syncPackageIndexBatch.ts]: fixed f6de5fb

**Review by ctizen — APPROVED** [review]

**Pull request merged**